### ISSUE TITLE
[Bug]: Pod Install fails with Custom Flags

### ISSUE THREAD

**Issue by K-Leon** [OP-Engineering/op-sqlite#183]
### What happened?

Using this SQLITE Flag works on Android perfectly fine:
```
  "op-sqlite": {
	  "sqliteFlags": "-DSQLITE_TEMP_STORE=2"
  },

```
Pod Install on IOS throws an error.

```
@op-engineering/op-sqlite/op-sqlite.podspec:163
 #  -------------------------------------------
 #      log_message.call("[OP-SQLITE] Custom SQLite flags: #{sqlite_flags}")
 >      xcconfig[:OTHER_CFLAGS] += " #{sqlite_flags}"
 #    end
 #  -------------------------------------------
```
I couldn't figure out what is wrong. 
Thank you very much for your hard work on this plugin!

### op-sqlite version

9.3.0

### React Native version

0.76.1

### Reproducible Example

-

**Comment by ospfranco** [comment]
Yeah, I realized this might be failing the other day. You will have to wait for a bit. I will take a look when I can but it might take a while

**Comment by K-Leon** [comment]
No worries - I'm glad I didn't mess it u p :)
Thank you very much!

**Comment by ospfranco** [comment]
This should be fixed in the released version: `10.0.0-tokenizers-beta8`. Give it a try there and let me know if it works.

**Comment by ospfranco** [comment]
Released

https://github.com/OP-Engineering/op-sqlite/releases/tag/10.1.0
